.github/PR_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thank you for contributing to Harbor!
+
+### Summary of your change
+
+
+### Issue being fixed
+
+Fixes #(issue)
+
+### Please indicate you've done the following:
+
+- [ ] [Accepted the DCO](https://github.com/goharbor/harbor/blob/master/CONTRIBUTING.md#commit). Commits without the DCO will delay acceptance.
+- [ ] Updated the corresponding documentation in this repo or https://github.com/goharbor/website.
```

Comment:
Thanks Abbie for creating the template. Assuming this one is for those PRs need document change. If the PR doesn't need document change, how submitter make the choice. And it would be better to change the document after this PR is merged.
One suggestion is to make it as a note only.

Reply:
Thats a good point. My first goal is to remind folks about updating the docs. I think my concern with making it a note, instead of a check mark is that there is no sign-off from a contributor.

What if we made it more open ended? something like,

```- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed```

My concern is that if we merge a PR without an option like this as a part of the checklist, then there will be no way to remember to open and docs issue or PR after the pr is merged. I'm interested to hear thoughts and feedback on this idea

Reply:
LGTM